.github/workflows/pr_tests.yml

```diff
@@ -126,47 +126,25 @@ jobs:
       #     sudo apt-get update
       #     sudo apt-get install libavdevice-dev libavfilter-dev libopus-dev libvpx-dev pkg-config
 
-      - name: Install PyTorch Linux and MacOS
-        if: startsWith(runner.os, 'Windows') != true
+      - name: Adjust PyTorch versions
         run: |
-          ./scripts/pytorch_install.sh ${{ matrix.torch-version }}
-
-      - name: Install PyTorch Windows
-        if: startsWith(runner.os, 'Windows')
-        run: |
-          ./scripts/pytorch_install.ps1 ${{ matrix.torch-version }}
+          python .github/adjust_torch_versions.py ./requirements.txt ${{ matrix.torch-version }}
 
       - name: Install packages
         run: |
-          pip install -r requirements.txt --default-timeout=60
```

Comment:
We had timeout issues with pip, are you sure the job task timeout will do the same thing?

Reply:
in combination with `continue-on-error: true` it should
but if you silently accept missing some packages isn't it making the test weak, imagine that none of the packages is installed, you skip them in testing and show as all pass... so you think all is fine, but in fact, you just don't know...